From e5ad75e3ee17331ddba32c9133eae6bbdac3fbb6 Mon Sep 17 00:00:00 2001
From: duli <duli4868@gmail.com>
Date: Wed, 2 Oct 2024 18:26:56 +0800
Subject: [PATCH] fix: go test ld undefined error by remove clickhouse-go

github.com/segmentio/asm build must need go module feature, disable it
will cause ld error, due to clickhouse-go is not in use. we delete it to
fix that.
---
 vendor/github.com/go-testfixtures/testfixtures/v3/clickhouse.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/vendor/github.com/go-testfixtures/testfixtures/v3/clickhouse.go b/vendor/github.com/go-testfixtures/testfixtures/v3/clickhouse.go
index c01c9ee6..7f6c76a8 100644
--- a/vendor/github.com/go-testfixtures/testfixtures/v3/clickhouse.go
+++ b/vendor/github.com/go-testfixtures/testfixtures/v3/clickhouse.go
@@ -4,7 +4,6 @@ import (
 	"database/sql"
 	"fmt"
 
-	_ "github.com/ClickHouse/clickhouse-go/v2"
 )
 
 type clickhouse struct {
-- 
2.46.2

